class Error(Exception):
    print('ERROR!')
    pass


class Warning(Warning):
    print('WARNING!')
    pass